h2o-core/src/test/java/water/TestUtil.java

```diff
@@ -683,10 +688,20 @@ public static void assertVecEquals(Vec expecteds, Vec actuals, double delta) {
   }
 
   public static void assertVecEquals(String messagePrefix, Vec expecteds, Vec actuals, double delta) {
+    assertVecEquals(messagePrefix, expecteds, actuals, delta, false);
+  }
+
+  public static void assertVecEquals(String messagePrefix, Vec expecteds, Vec actuals, double delta, boolean relativeDelta) {
```

Comment:
Another way to do this would be to allow specifying both absolute and relative deltas and use both if both specified or just one of them if one of them is NaN.

Reply:
I find this approach more user-friendly and I cannot imagine a case when you would need both

Reply:
this would make it consistent with Model#testJavaScoring: double rel_epsilon, double abs_epsilon

for regression problems you might want to bind the absolute error as well as the relative error